spec/takeaway_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'takeaway'
+
+RSpec.describe Takeaway do
+
+  context "#show_menu" do
+    it "should print the menu" do
+      expect(subject.show_menu).to be_kind_of Menu
+    end
+  end
+
+  context "#place_order" do
+    it "should return a hash of quantity and Dish instance" do
+      expect(subject.place_order(5, "Cheeseburger")).to eq(33.95)
+    end
+
```

Comment:
I also did this in my program but I think there must be a better way to test the Place_Order method. As it stands, for this test to pass, both the place_order method & the update_total_cost method must work. This test therefore is testing 2 methods in 1 test (so if it fails, that could be because update_total_cost is broken or it could be because place_order is broken)